deps/check_latest_v8_version.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+# Script to compare current v8 version installed and the latest stable v8 version
+# Returns 1 if the versions are the same
+# Returns 0 if the latest stable version is higher than the current installed one
+
+import requests
+import sys
+
+# api-endpoint
+URL = "https://omahaproxy.appspot.com/all.json?os=linux&channel=stable"
```

Comment:
Perhaps we should fetch all OS versions, filter the OSes that we are building for in python (linux, win64 and mac) and only upgrade only automatically upgrade when all the versions are the same.  If they have held off on promoting the version to stable for one of the OSes because of some problem, then this might help us to avoid it.

However, this might be fine for now.

Reply:
I've never seen this happen in practice, especially for mac and linux.